packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.tsx

```diff
@@ -271,19 +296,23 @@ export class AtomicCommerceSearchBox
     this.suggestionManager.forceUpdate();
   }
 
-  public componentWillRender() {
+  private registerSearchboxSuggestionEvents() {
     this.searchBoxSuggestionEventsQueue.forEach((evt) => {
       this.suggestionManager.registerSuggestionsFromEvent(
         evt,
         this.suggestionBindings
       );
     });
-    this.searchBoxSuggestionEventsQueue = [];
+  }
+
+  public componentDidLoad() {
+    this.registerSearchboxSuggestionEvents();
+    this.hasRegisterSuggestions = true;
   }
```

Comment:
We need to record this a tad bit later. On the flip side, we put a fuse to ensure the default stuff ain't loaded (which would not have been necessary if stencil was using slot & template like native web-component does :( )